### PR TITLE
Add grpcurl manifest

### PR DESCRIPTION
**Pull request by juliaogris**

### CODE REVIEW

File: grpcurl.hcl [hunk 1]
```diff
@@ -0,0 +1,17 @@
+description = "Inspect and call gRPC services"
+test = "grpcurl --version"
+binaries = ["grpcurl"]
+
+darwin {
+  source = "https://github.com/fullstorydev/grpcurl/releases/download/v${version}/grpcurl_${version}_osx_${xarch}.tar.gz"
```
Comment (alecthomas) [inline grpcurl.hcl:6]: Alas there is no arm64 Darwin build. Can you `s,${arch},x86_64,` so it will work on M1 Macs?

**Review by camh- — COMMENTED** [review]
LGTM, but I don't know what `auto-version` does and is not really documented at https://cashapp.github.io/hermit/packaging/schema/auto-version/

File: grpcurl.hcl (hunk 1 above)
Comment (camh-) [reply grpcurl.hcl:6]: It doesn't look like there are arm64 releases: https://github.com/fullstorydev/grpcurl/releases/

Comment (camh-) [reply grpcurl.hcl:6]: my bad. i read the opposite of what you were saying. ignore me.

Comment (juliaogris) [reply grpcurl.hcl]: fixed, apologies I didn't see this.

**Pull request merged**